asistente_ladm_col/logic/quality_rules/qr_parcel_department_field_valid_code.py

```diff
@@ -0,0 +1,96 @@
+"""
+/***************************************************************************
+                              Asistente LADM-COL
+                             --------------------
+        begin           : 2022-06-12
+        git sha         : :%H$
+        copyright       : (C) 2022 by Sergio Ramírez (SwissTierras Colombia)
+        email           : seralra96@gmail.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4003,
+                                                           QRE_IGACR4003E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRParcelWithInvalidDepartmentCode(AbstractLogicQualityRule):
+    """
+    Check that parcels have incorrect department code
+    """
+    _ERROR_01 = QRE_IGACR4003E01  # Parcel with incorrect department code
+
+    def __init__(self):
+        AbstractLogicQualityRule.__init__(self)
+
+        self._id = QR_IGACR4003
+        self._name = "Revisar que el código de departamento de la tabla predio tiene dos caracteres numéricos"
```

Comment:
To shorten the name of the quality rule, do you think it is a good idea to rename the rule to: "El código de departamento de la tabla predio tiene dos caracteres numéricos"?

Reply:
Fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/548/commits/9d6d30fc5294cba38c1a628da5c7479f830e2c3e